client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -203,6 +203,13 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
                 position: 'right',
                 tooltip: 'The number of repositories that have been cloned.',
             },
+            {
+                value: data.repositoryStats.indexed,
+                description: 'Indexed',
+                color: 'var(--body-color)',
+                position: 'right',
+                tooltip: 'The number of repositories that have been index for search.',
```

Comment:
Small typo correction :)

```suggestion
                tooltip: 'The number of repositories that have been indexed for search.',
```